kansha/card/comp.py

```diff
@@ -67,6 +52,8 @@ def __init__(self, id_, column, services_service, data=None):
         self.column = column
         self._services = services_service
         self._data = data
+        self.card_extensions = tuple()
+        self.card_repo = card_extensions
```

Comment:
1. `card_extensions` -> `extensions` (we are in the card so we know `extensions` refer to the extensions of this card)
2. `card_repo` -> `card_extensions` (we keep the same name than the parameter)
3. `tuple()` -> `()` (use the language syntaxe to create the object, not a call to their types)